@here/harp-mapview/lib/MapView.ts

```diff
@@ -2355,25 +2353,6 @@ export class MapView extends THREE.EventDispatcher {
         this.clearTileCache();
     }
 
-    /**
-     * Allows the user to block some areas of the screen.
-     * @note To clear, simply pass in an empty array.
-     * @param screenSpaceBoxes Boxes that are in screen space.
-     * @deprecated
-     */
-    setLabelAvoidanceWithBoxes(screenSpaceBoxes: Math2D.Box[]) {
```

Comment:
What is new API for this functionality ?
For lines, we can create GeoJSON with new technique, what about boxes.

I see this as very useful API, not sure why removing it.

Reply:
This was an API I added previously, unfortunately it had a bid snow ball affect on the inner code. It meant that, because the current ScreenCollisions class uses an rtree which is basically the NDC value scaled by height / width, that I had to convert from screen coordinates to this scaled NDC space (this is used, because it is the space generated by the project function in the ScreenProjector class).

By removing this API (which BTW was added as @deprecated), I can avoid this internal shift and was able to see an improvement in performance also.

Reply:
Ok, fair enough. 
But how is this PR is related to blocking lines which are only remotely related to blocking of whole areas.